templates/frontend/components/header.tpl

```diff
@@ -71,17 +71,17 @@
 				</div>
 
 				{* Primary site navigation *}
-				{if $currentContext}
-					<nav class="pkp_navigation_primary_row" aria-label="{translate|escape key="common.navigation.site"}">
-						<div class="pkp_navigation_primary_wrapper">
-							{* Primary navigation menu for current application *}
-							{load_menu name="primary" id="navigationPrimary" ulClass="pkp_navigation_primary"}
+				<nav class="pkp_navigation_primary_row" aria-label="{translate|escape key="common.navigation.site"}">
+					<div class="pkp_navigation_primary_wrapper">
+						{* Primary navigation menu for current application *}
+						{load_menu name="primary" id="navigationPrimary" ulClass="pkp_navigation_primary"}
 
+						{if $currentContext}
 							{* Search form *}
 							{include file="frontend/components/searchForm_simple.tpl"}
-						</div>
-					</nav>
-				{/if}
+						{/if}
+					</div>
+				</nav>
```

Comment:
This will leave empty HTML elements if no navigation menu is assigned, which will be common on the site level since there's no default nav menu item. Can you capture the menu and check if it's empty?

Off the top of my head that would look something like this:

```
{capture assign="primaryMenu"}{load_menu ...}{/capture}
{if !empty(trim($primaryMenu))}
  ...
{/if}
```